server/calabash-js/src/calabash.js

```diff
@@ -1,14 +1,14 @@
-(function () {
+(function() {
     /** David Mark's isHostMethod function,
-      * http://peter.michaux.ca/articles/feature-detection-state-of-the-art-browser-scripting
-      * Modified to use strict equality
-      */
+     * http://peter.michaux.ca/articles/feature-detection-state-of-the-art-browser-scripting
+     * Modified to use strict equality
+     */
     function isHostMethod (object, property)
     {
-      var t = typeof object[property];
-      return t==='function' ||
-             (!!(t==='object' && object[property])) ||
-             t==='unknown';
+        var t = typeof object[property];
+        return t==='function' ||
+            (!!(t==='object' && object[property])) ||
+            t==='unknown';
```

Comment:
Is this a purely whitespace change?  I know it is a pain, but I would have preferred:

1. Commit the whitespace changes on the main branch and push
2. Rebase the your local branch (before pushing)